src/components/header/mobile-menu/mobile-menu.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from "react";
+import styles from "./mobile-menu.module.scss";
+import { ReactComponent as MenuLine } from "../../../assets/images/Line.svg";
+
+export const MenuMobile = () => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const toggleMenu = () => {
+    setIsOpen(!isOpen);
+  };
+
+  return (
+    <div className={styles.mobileMenu}>
+      <div className={styles.menuButton} onClick={toggleMenu}>
+        <MenuLine />
+      </div>
+      {isOpen && (
+        <nav className={styles.menuItems}>
+          <ul>
+            <li>
+              <a href="/login">Login</a>
+            </li>
+            <li>
+              <a href="/register">Register</a>
+            </li>
+          </ul>
+        </nav>
+      )}
```

Comment:
avoid short circuit operators here because for some simple types first check can be rendered as text on the screen. 

https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx

Best practice is: 

```tsx
{isOpen ? <div></div> : null}
```

to be adusted across the whole app

Reply:
ok, now I understand. Changed across the whole app